File: generated-sources/scalaz/mojang-sessions/src/main/scala/org/openapitools/client/api/PlayerProfileTexturePropertyValue.scala
package org.openapitools.client.api

import argonaut._
import argonaut.EncodeJson._
import argonaut.DecodeJson._

import org.http4s.{EntityDecoder, EntityEncoder}
import org.http4s.argonaut._
import org.joda.time.DateTime
import PlayerProfileTexturePropertyValue._

case class PlayerProfileTexturePropertyValue (
  /* UNIX timestamp in milliseconds */
  timestamp: Long,
/* The player UUID without hyphens */
  profileId: String,
/* The player's name */
  profileName: String,
/* Only present if unsigned was set to false in the request */
  signatureRequired: Option[Boolean],
textures: Option[PlayerTexture])

object PlayerProfileTexturePropertyValue {
  import DateTimeCodecs._

  implicit val PlayerProfileTexturePropertyValueCodecJson: CodecJson[PlayerProfileTexturePropertyValue] = CodecJson.derive[PlayerProfileTexturePropertyValue]
  implicit val PlayerProfileTexturePropertyValueDecoder: EntityDecoder[PlayerProfileTexturePropertyValue] = jsonOf[PlayerProfileTexturePropertyValue]
  implicit val PlayerProfileTexturePropertyValueEncoder: EntityEncoder[PlayerProfileTexturePropertyValue] = jsonEncoderOf[PlayerProfileTexturePropertyValue]
}

File: generated-sources/php-slim/mojang-sessions/lib/com/github/asyncmc/mojang/sessions/php/slim/model/PlayerProfileTexturePropertyValue.php
<?php
/**
 * PlayerProfileTexturePropertyValue
 */
namespace com.github.asyncmc.mojang.sessions.php.slim.server\com.github.asyncmc.mojang.sessions.php.slim.model;

/**
 * PlayerProfileTexturePropertyValue
 */
class PlayerProfileTexturePropertyValue
{
    
    /** @var int $timestamp UNIX timestamp in milliseconds*/
    private $timestamp;
    
    /** @var string $profileId The player UUID without hyphens*/
    private $profileId;
    
    /** @var string $profileName The player&#39;s name*/
    private $profileName;
    
    /** @var bool $signatureRequired Only present if unsigned was set to false in the request*/
    private $signatureRequired;
    
    /** @var \com.github.asyncmc.mojang.sessions.php.slim.server\com.github.asyncmc.mojang.sessions.php.slim.model\PlayerTexture $textures */
    private $textures;
}
